idiss/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.6.1
+  - Add dependency to feature `ffi` from `concordium_base` such that `ffi_helpers` module can be used.
```

Comment:
It is better to be more direct by stating that the `free_array_len_cap` is now exported from the shared library.